src/tests.c

```diff
@@ -4009,22 +4009,19 @@ static void test_add_neg_y_diff_x(void) {
      * which this test is a regression test for.
      *
      * These points were generated in sage as
-     * # secp256k1 params
-     * F = FiniteField (0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEFFFFFC2F)
-     * C = EllipticCurve ([F (0), F (7)])
-     * G = C.lift_x(0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798)
-     * N = FiniteField(G.order())
+     *
+     * load("secp256k1_params.sage")
      *
      * # endomorphism values (lambda is 1^{1/3} in N, beta is 1^{1/3} in F)
```

Comment:
comment needs to be updated, I think

Reply:
Done (assumed that you only meant the rename from N to Z).